opentracing/tracecontext.go

```diff
@@ -39,12 +39,16 @@ type TraceContext interface {
 	// TraceContext, and that can add up to a lot of network and cpu
 	// overhead.
 	//
+	// IMPORTANT NOTE #3: Trace tags are case-insensitive: implementations may
```

Comment:
this is where my comment meant to be wrt lc transmission. Impact is that case preservation of keys is unsupported.